packages/oc-gs-storage-adapter/__test__/gs.test.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+const gs = require('../');
+const Storage = require('@google-cloud/storage');
+
+// //Mock AWS functions
```

Comment:
We reworked and improved the tests of the oc-s3-storage-adapter, perhapsh is worth to sync [those](https://github.com/opencomponents/storage-adapters/blob/master/packages/oc-s3-storage-adapter/__test__/s3.test.js) ones?